### PR TITLE
New Feature: webpage now displays <img> of SVG

### PR DESCRIPTION
**Pull request by Thomas-Boi**
# **New feature section**

## Double check these details before you open a PR

- [x] PR does not match another non-stale PR currently opened
- [x] PR name matches the format *New Feature: a brief description of feature*

## This PR adds/fixes
Adds a section to display the `` of the selected SVG to the website. 

This offers users an option to link external SVGs rather than using inline/pure SVGs. This will change dynamically depending on the currently selected SVGs.

I've tested the feature and here's what it looks like:

image

image

The feature above was inspired by [this site](https://renevds.github.io/language-icon-widget-pages/#/) which was submitted by @renevds on Discord.

Also redo the icon section and add a hint on how to increase the icon's size:
image

## Notes

Later on, I'll work on a copy button to make things easier for the user. That'll be in a separate PR.

### CODE REVIEW

**Comment by renevds** [comment]
Looks awesome :)
Small note, adding a width property slider would be great, standard these are really big.

File: docs/index.html [hunk 1]
```diff
@@ -66,13 +66,16 @@ <h4>Font versions <input type='color' value='#60be86' ng-model="fontBackground">
               </li>
             </div>
           </ul>
+          <p>Put this in your header</p>
```
Comment (Panquesito7) [inline docs/index.html:69]:
```suggestion
          <p>Place this in your header</p>
```

**Review by Panquesito7 — COMMENTED** [review]
Overall, everything else looks great! Awesome work, @Thomas-Boi! 😄👍

**Comment by Thomas-Boi** [comment]
> Looks awesome :)

Thank you 😄. It's all inspired by your idea.
> Small note, adding a width property slider would be great, standard these are really big.

I think that's a bit of an overkill. I'll leave a note telling people that they can change the width and height to get what they want though.

**Review by Panquesito7 — APPROVED** [review]
LGTM 🚀 Thank you for contributing, @Thomas-Boi and @renevds! 😄👍

**Pull request merged**